vm/src/builtins/complex.rs

```diff
@@ -545,13 +545,17 @@ fn parse_str(s: &str) -> Option<Complex64> {
     };
 
     let value = match s.strip_suffix(|c| c == 'j' || c == 'J') {
-        None => Complex64::new(crate::literal::float::parse_str(s)?, 0.0),
+        None => {
+            let stripped = float::float_strip_underscores(s.as_bytes())?;
+            Complex64::new(crate::literal::float::parse_bytes(&stripped)?, 0.0)
```

Comment:
is there any case parse_bytes is called without float_strip_underscores?
How about put it in parse_bytes if it is necessary?

Reply:
I did not come across any cases where parse_bytes calls needed to or could be performed without underscore handling. I'll look into putting this directly in parse_bytes :+1: